azurepipelines/e2e_test/scenarios/testingtoolkit/_adu_test_toolkit.py

```diff
@@ -53,11 +53,9 @@ def ParseFromOsEnviron(self):
         self._aduEndpoint = os.environ['ADU_ENDPOINT']
         self._aduInstanceId = os.environ['ADU_INSTANCE_ID']
         self._iotHubUrl = os.environ['IOTHUB_URL']
-        self._iotHubConnectionString = os.environ['IOTHUB_CONNECTION_STRING']
-        self._aadRegistrarClientId = os.environ['AAD_REGISTRAR_CLIENT_ID']
-        self._aadRegistrarTenantId = os.environ['AAD_REGISTRAR_TENANT_ID']
-        self._aadRegistrarClientSecret = os.environ['AAD_CLIENT_SECRET']
-
+        self._msiTenantId = os.environ['AZURE_TENANT_ID']
```

Comment:
Are these the ones representing the MSI we created for the Pool? It would make sense that this is the credential that is actually used behind the scenes when running the command.